PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -493,11 +493,11 @@ class HTTPClientTests: XCTestCase {
 
         let totalRequests = Int.random(in: 50..<100)
         for requestNumber in 0..<totalRequests {
-            self.client.performPOSTRequest(serially: true,
-                                           path: path,
-                                           requestBody: ["requestNumber": requestNumber],
-                                           headers: [:]) { (status, data, error) in
-                completionCallCount += 1
+            completionCallCount += 1
+            client.performPOSTRequest(serially: true,
```

Comment:
I think there *might* be a race condition here, we should increment the count before firing the post request anyway.

Reply:
Yeah, good point. Will revert that one